person.rb

```diff
@@ -0,0 +1,28 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def _id
+    @id
+  end
+
+  def _name
+    @name
+  end
+
+  def _age
+    @age
+  end
```

Comment:
Not only do you need getters for these variables, but you also need setters as well, please set them up. Consider using the `attr_accessor` and `attr_reader` to set up your setters and getters.